src/nvim/window.c

```diff
@@ -487,6 +487,16 @@ do_window (
   }
 }
 
+static void cmd_with_count(char *cmd, char_u *bufp, size_t bufsize,
+                           uint32_t Prenum)
+{
+  size_t len = xstrlcpy((char *)bufp, cmd, bufsize);
+
+  if (Prenum > 0 && len < bufsize) {
+    vim_snprintf((char *)bufp + len, bufsize - len, "%ld", Prenum);
```

Comment:
Use a portable format specifier (like the old Neovim version, here: https://github.com/neovim/neovim/pull/2041/files#diff-40d9eb21ee75b3df148c827ec4d4310aL129 ). `Prenum` is `long` so `PRId64` is the format specifier for that. E.g.:

```c
  if (Prenum > 0 && len < bufsize) {
    vim_snprintf((char *)bufp + len, bufsize - len, "%" PRId64, Prenum);
```

Why does `cmd_with_count()` declare `Prenum` as `uint32_t` instead of `long`?


See also: https://github.com/neovim/neovim/pull/1729#discussion_r22436056